controllers/base-controller.go

```diff
@@ -293,8 +330,8 @@ func (this *baseController) redirect(webRedirectDir string) {
 	)
 }
 
-func (this *baseController) setCookies(value map[string]string) {
+func (this *baseController) setCookies(value map[string]string, isSensitive bool) {
 	for k, v := range value {
-		this.Ctx.SetCookie(k, v, "3600", "/")
+		this.Ctx.SetCookie(k, v, "3600", "/", "", true, isSensitive)
```

Comment:
this.Ctx.SetCookie(k, v, 3600, "/", "", true, isSensitive)

Reply:
???   , is ok